src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperScopeManager.cs

```diff
@@ -31,18 +32,36 @@ public TagHelperScopeManager()
         /// <param name="executeChildContentAsync">A delegate used to execute the child content asynchronously.</param>
         /// <param name="startWritingScope">A delegate used to start a writing scope in a Razor page.</param>
         /// <param name="endWritingScope">A delegate used to end a writing scope in a Razor page.</param>
+        /// <param name="parentExecutionContext">The parent <see cref="TagHelperExecutionContext"/>.</param>
```

Comment:
nit: suggest placing this parameter above the various `Func` and `Action` parameters, as you've done in the `TagHelperContext` constructor